sentry/src/main/java/io/sentry/TransactionPerformanceCollector.java

```diff
@@ -7,8 +7,12 @@
 
 public interface TransactionPerformanceCollector {
```

Comment:
I guess we could rename it to `PerformanceMetricsCollector` or something since it's not bound to transactions anymore? We'd probably need something similar for replays in the future, so I guess it's better to rename it now since it's part of v8?

Reply:
Or maybe `AggregatedPerformanceCollector`/`CompositePerformanceCollector` since it combines all other `IPerformanceCollectors`

Reply:
I renamed it to `CompositePerformanceCollector`